src/encoded/audit/genetic_modification.py

```diff
@@ -3,6 +3,21 @@
     audit_checker,
 )
 
+
+@audit_checker('GeneticModification', frame=['modified_site_by_target_id'])
+def audit_genetic_modification_target(value, system):
+    '''
+    Genetic modifications should targeting (modified_site_by_target_id) a wild
+    type gene, i.e. a target without modifications, as defined in the schema.
+    '''
+    if value.get('modified_site_by_target_id', {}).get('modifications'):
+        detail = ('Genetic modification {} is targeting {} which has already '
```

Comment:
I'm just using this as an example but there are mixed code formatting styles all over the place.  
```
        detail = (
            'Genetic modification {} is targeting {} which has already '
            'been modified.'.format(
                value['@id'], 
                value['modified_site_by_target_id']['@id'],
            )
        )
```
this style if more widely applicable for various use cases.  
-End parens/brackets dropped to the next line
-if you drop one arg/item to the next line the put them all in a line with trailing commas
-pulling stuff to the left makes more readable code and allows for longer lines

We don't adhere to this, so just FYI.  There are many styles, but we should try not to mixed them.